### PR TITLE
Chore: LocalAdapter path + test

### PR DESCRIPTION
**Pull request by stefanorosanelli**
This PR fixes default local adapter path configuration using `webroot/_files` as default and avoiding double slashes `//`

Bonus track: temporary folders created in testcase are removed after test execution.

### CODE REVIEW

**Review by didoda — APPROVED** [review]

File: plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ public function initialize(array $config)
     {
         $success = parent::initialize($config);
 
-        if (empty($this->_config['baseUrl']) && strpos($this->getConfig('path'), WWW_ROOT . DS) === 0) {
```
Comment (fquffio) [inline plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php:47]: The trailing directory separator was there to avoid errors in case there's another directory whose path begins with `BEDITA_ROOT/webroot`. For instance, if i want to store my static files under `BEDITA_ROOT/webroot2` the automatic detection of base URL should fail. However this is quite an edge case, and we may want to ignore it.

File: plugins/BEdita/Core/tests/TestCase/Filesystem/Adapter/LocalAdapterTest.php [hunk 2]
```diff
@@ -23,6 +23,24 @@
  */
 class LocalAdapterTest extends TestCase
 {
+    /**
+     * Temporary test files path, removed after test
+     * @var string
+     */
+    const FILES_PATH = WWW_ROOT . 'static-files' . DS . 'subdir';
```
Comment (fquffio) [inline plugins/BEdita/Core/tests/TestCase/Filesystem/Adapter/LocalAdapterTest.php:30]: In `tests/bootstrap.php` we configure `LocalAdapter` to use `plugins/BEdita/Core/tests/uploads` as its root. That configuration is not used in this case, but we might use the same path…

**Review by fquffio — APPROVED** [review]

File: plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php [hunk 3]
```diff
@@ -44,7 +44,7 @@ public function initialize(array $config)
...
+        if (empty($this->_config['baseUrl']) && strpos($this->getConfig('path'), WWW_ROOT) === 0) {
             // Files are stored within the document root, so base URL can be automatically detected if not already set.
             $path = str_replace(DS, '/', substr($this->getConfig('path'), strlen(WWW_ROOT)));
```
Comment (fquffio) [inline plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php:49]: ...instead, I'd put `strlen(WWW_ROOT) + 1` or `strlen(WWW_ROOT . DS)` here.

File: plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php (hunk 1 above)
Comment (stefanorosanelli) [reply plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php:47]: You're right but `WWW_ROOT` is defined as `ROOT . DS . 'webroot' . DS` so it already contains dir separator the end.

File: plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php (hunk 3 above)
Comment (stefanorosanelli) [reply plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php:49]: also not necessary since `WWW_ROOT` contains last dir separator

File: plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php (hunk 1 above)
Comment (fquffio) [reply plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php:47]: Oh, right. Sorry, I hadn't noticed that.
